build.gradle

```diff
@@ -17,7 +17,7 @@ apply plugin: 'java'
 apply plugin: 'eclipse' // Eclipse integration
 
 description = """"""
-def version_str = "1.0.0"
+def version_str = "1.0.1"
```

Comment:
Why not have it read the latest git tag instead?

Reply:
Because we don't always want to create the latest version, and binding the version directly in the commits is faster to us.